### PR TITLE
Updated another way of string translation in js

### PR DESCRIPTION
**Pull request by srsathish92**
## Purpose of this pull request

This pull request (PR) to update the another way of strings translation in js.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/translations/translate_theory.html#add_strings_js
- https://devdocs.magento.com/guides/v2.4/frontend-dev-guide/translations/translate_theory.html#add_strings_js

## Links to Magento source code
- https://github.com/magento/magento2/blob/2.4/app/code/Magento/Checkout/view/frontend/web/js/view/shipping.js#L91

### CODE REVIEW

File: src/guides/v2.3/frontend-dev-guide/translations/translate_theory.md [hunk 1]
```diff
@@ -125,20 +125,28 @@ To ensure that the text you add in a `.js` file is collected by the i18n tool an
...
-   define (['jquery', 'mage/translate'], function ($) {...});
+   define (['jquery', 'mage/translate'], function ($, $t) {...});
    ```
 
 1. Use the `$.mage.__('')` function when adding a string:
 
    ```javascript
    $.mage.__('<string>');
    ```
+   (or)
```
Comment (BarnyShergold) [inline src/guides/v2.3/frontend-dev-guide/translations/translate_theory.md:136]: Braces around a textual word look a little odd and may cause confusion
```suggestion
   or
```

File: src/guides/v2.3/frontend-dev-guide/translations/translate_theory.md [hunk 2]
```diff
@@ -125,20 +125,28 @@ To ensure that the text you add in a `.js` file is collected by the i18n tool an
...
+   ```javascript
+   $t('<string>');
+   ```
 
    If your string contains a variable, to add a placeholder for this variable to the string stored in the dictionary, use the syntax similar to the following:
 
    ```javascript
    $.mage.__('Hello %1').replace('%1', yourVariable);
    ```
+   (or)
```
Comment (BarnyShergold) [inline src/guides/v2.3/frontend-dev-guide/translations/translate_theory.md:146]:
```suggestion
   or
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Nice addition

**Review by BarnyShergold — APPROVED** [review]
Thanks!

**Comment by srsathish92** [comment]
@BarnyShergold Thank you for review

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Pull request merged**